compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/environment/EnvironmentPackageCache.java

```diff
@@ -81,4 +81,4 @@ public List<Package> getPackages(PackageOrg packageOrg, PackageName packageName)
     public void removePackage(PackageId packageId) {
         projects.remove(packageId);
     }
-}
+}
```

Comment:
The file should end with a newline. This also, can be captures when running the checkstyle task.